sdk/fileapi_sdk/Common/fileapi_sdk_global.h

```diff
@@ -0,0 +1,10 @@
+#ifndef FILEAPI_SDK_GLOBAL_H
+#define FILEAPI_SDK_GLOBAL_H
+
+#if defined(FILEAPI_SDK_LIBRARY)

```

Comment:
This file is auto generated. i can remove it if we don't need it

---
In reply to: [134200877](https://github.com/AzureAD/rms-sdk-for-cpp/pull/132#discussion_r134200877) [](ancestors = 134200877)